vmdb/Gemfile

```diff
@@ -9,16 +9,15 @@ MiqBundler.include_gemfile("../lib/Gemfile", binding)
 
 gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
 gem 'angularjs-rails', '=1.2.4'
-gem 'jquery-rails', "~>2.2.2"
+gem 'jquery-rails', "=2.2.2"
```

Comment:
Bootstrap requires this version or above and I wanted to minimize the problems with @romanblanco's ongoing JS refactoring. After he's done, we should consider a bigger upgrade with no version locking.